taxonomies/categories.txt

```diff
@@ -42268,6 +43020,7 @@ wikidata:en:Q1365891
 
 <en:Gherkins
 en:Pickled gherkins
+de:Gewürzgurken,Essiggurken,Cornichons,Eingelegte Gurken
```

Comment:
`ERROR - de:essiggurken already is a synonym of essiggurken (en:gherkins) - cannot make a synonym of gewürzgurken for en:pickled-gherkins`

`ERROR - de:eingelegte-gurken already is a synonym of eingelegte-gurken (en:pickled-cucumbers) - cannot make a synonym of gewürzgurken for en:pickled-gherkins`